jenkins/clouds/aws/daily/policies/run_policies.py

```
@@ -148,3 +151,43 @@ def run_policies(policies: list, dry_run: str = 'yes'):
+        os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))))
+    ORION_CONFIG_PATH = os.path.join(REPO_ROOT, 'orion-configs', 'cg-policy-regressions.yaml')
+    ORION_TEST_NAME = 'cg-policy-regressions'
+
+    run_cmd("echo Running Orion metrics rollup")
+    run_cmd(
+        f"""podman run --rm --name cloud-governance --net="host" -e account="{account_name}" -e policy="orion_metrics_rollup" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}" -e es_user="{ES_USER}" -e es_password="{ES_PASSWORD}" -e log_level="INFO" {QUAY_CLOUD_GOVERNANCE_REPOSITORY}""")
+
+    es_scheme = 'https' if str(ES_PORT) == '443' else 'http'
+    es_auth = f'{ES_USER}:{ES_PASSWORD}@' if ES_USER else ''
+    es_server = f'{es_scheme}://{es_auth}{ES_HOST}:{ES_PORT}'
+    orion_output_base = f'/tmp/orion-output-{account_name}.json'
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/orion-output-"

(S108)